configs/components/echam/echam.yaml

```diff
@@ -529,6 +497,24 @@ choose_wiso_code:
                                 add_coupling_fields:
                                         "[[wiso_fields-->FIELD]]":
                                                 grid: atmo
+                                add_streams:
+                                        - wiso
+                                        - accw_wiso
+                                        - sf_wiso
+                                        - rad
+                                add_forcing_files:
```

Comment:
I am not sure here: is the ozone[a,b,c] specific to wiso?

Reply:
me neither, but it was only under the version `6.3.05p2-wiso`.